mycroft/skills/common_play_skill.py

```diff
@@ -165,6 +165,8 @@ def __handle_play_start(self, message):
         # "... on the chromecast"
         self.play_service_string = phrase
 
+        self.bus.emit(Message('active_skill_request',  {'skill_id': self.skill_id}))
```

Comment:
im late to the party, but theres a MycroftSkill method that handles this `self.make_active()`

Reply:
It was getting called inconsistently based on match types. I deprecated it in the next PR